### PR TITLE
Fix docker provider install script to account for both 17.* and 1.* v…

### PR DESCRIPTION
**Pull request by samisms**
Fix docker provider install script to account for both 17.* (new docker-ee and docker-ce) and 1.1*  versionings 
@Microsoft/omsagent-devs @keikhara

### CODE REVIEW

File: installer/oss-kits/docker-oss-test.sh [hunk 1]
```diff
@@ -21,4 +23,4 @@ if [ "$(uname -m)" == "x86_64" ]; then
 else
     echo "  Only 64-bit systems are supported. Docker agent will not be installed." 1>&2
     exit 1
-fi
\ No newline at end of file
+fi
```
Comment (lagalbra) [inline installer/oss-kits/docker-oss-test.sh]: nit: undo this change unless you got some error without redoing this line.

Comment (samisms) [inline installer/oss-kits/docker-oss-test.sh]: Artefact of vim. Just a new line. Should I still change it ?

Comment (lagalbra) [inline installer/oss-kits/docker-oss-test.sh]: I guess it doesn't really matter; this doesn't look like one of the scripts that's super picky.

Comment (samisms) [inline installer/oss-kits/docker-oss-test.sh]: I fixed it anyway. b5fae9e

**Comment by sugr4** [comment]
Please squash your commits to one single commit, and rebase to master before pushing and merging with master

File: installer/oss-kits/docker-oss-test.sh (hunk 1 above)
Comment (NarineM) [inline installer/oss-kits/docker-oss-test.sh]: It matters actually in Linux files. some tools still require empty line at the end of the file. Let us make sure we do not break that.
